CMakeLists.txt

```diff
@@ -88,6 +89,11 @@ else()
 endif()
 message( VERBOSE "AMDGPU_TARGETS=${AMDGPU_TARGETS}")
 
+if(NOT ${CURRENT_GPU} IN_LIST AMDGPU_TARGETS)
+  MESSAGE("Unsupported GPU; Cannot build tests/samples")
+  RETURN()
```

Comment:
Instead of returning, turn the flags for ROCWMMA_BUILD_TESTS and ROCWMMA_BUILD_SAMPLES to OFF. This will turn into a source install instead which is still valid. If you return here, no package information is created, and this is why the build is failing.